Src/AutoNSubstitute/AutoNSubstitute.csproj

```diff
@@ -64,6 +64,7 @@
   <ItemGroup>
     <Compile Include="AutoConfiguredNSubstituteCustomization.cs" />
     <Compile Include="AutoNSubstituteCustomization.cs" />
+    <Compile Include="AutoSubstituteCustomization.cs" />

```

Comment:
Thanks for the clarification. Here are the remaining changes I'd like to make.
- move code from the new `AutoSubstituteCustomization` class into the two existing  `AutoNSubstituteCustomization` and `AutoConfiguredNSubstituteCustomization` classes;
- eliminate the new `AutoSubistituteCustomization` class;
- rename the new `SubstituteAttribute` to `NSubstituteAttribute`;
- rename the new `SubstituteAttributeRelay` to `NSubsituteAttributeRelay`;
- rename the new `SubstituteBuilder` to `NSubstituteRequestHandler`;
- rename the new `SubstituteRelay` to `NSubstituteAbstractTypeRelay`;
- rename the new `SubstituteRequest` to `NSubstituteRequest`;
- `[Obsolete]` the existing `NSubstituteBuilder` which is being replaced by the new `NSubstituteAbstractTypeRelay`; 

Do you agree?